server/services/event.ts

```
@@ -235,8 +235,8 @@ export async function getEventBySlug(
   options: {
     lean?: boolean;
   } = {
-    lean: true,
-  },
+      lean: true,
```

ESLint

[error] Delete `··`

(prettier/prettier)